lib/private/Files/ObjectStore/ObjectStoreStorage.php

```diff
@@ -592,7 +592,13 @@ public function copyFromStorage(
 
 	public function moveFromStorage(IStorage $sourceStorage, string $sourceInternalPath, string $targetInternalPath, ?ICacheEntry $sourceCacheEntry = null): bool {
 		$sourceCache = $sourceStorage->getCache();
-		if ($sourceStorage->instanceOfStorage(ObjectStoreStorage::class) && $sourceStorage->getObjectStore()->getStorageId() === $this->getObjectStore()->getStorageId()) {
+		if (
+			$sourceStorage->instanceOfStorage(ObjectStoreStorage::class) &&
+			$sourceStorage->getObjectStore()->getStorageId() === $this->getObjectStore()->getStorageId()
```

Comment:
```suggestion
			$sourceStorage->instanceOfStorage(ObjectStoreStorage::class)
			&& $sourceStorage->getObjectStore()->getStorageId() === $this->getObjectStore()->getStorageId()
```

IIRC thats our codestyle?

Reply:
But it is not aligned :(